### PR TITLE
Fix uncropped image dimension attributes

### PR DESCRIPTION
**Pull request by roborourke**
A previous change introduced with WP 5.5 in Altis v5 meant that in some cases when an image was uncropped incorrect width and height attributes were added to the image.

### CODE REVIEW

**Review by jazzsequence — APPROVED** [review]
👍

File: inc/cropper/namespace.php [hunk 1]
```diff
@@ -996,9 +996,7 @@ function get_img_src_dimensions( string $image_src, array $image_meta ) {
 	} elseif ( isset( $tachyon_args['fit'] ) ) {
 		// Image is uncropped.
 		list( $width, $height ) = explode( ',', $tachyon_args['fit'] );
-		if ( empty( $height ) ) {
-			list( $width, $height ) = wp_constrain_dimensions( $image_meta['width'], $image_meta['height'], $width );
-		}
+		list( $width, $height ) = wp_constrain_dimensions( $image_meta['width'], $image_meta['height'], $width, $height ?? 0 );
```
Comment (goldenapples) [inline inc/cropper/namespace.php:999]: Drive-by comment: why are you redefining $width and $height? Should the previous line be removed?

Comment (jazzsequence) [reply inc/cropper/namespace.php:999]: Oh, good catch

Comment (roborourke) [reply inc/cropper/namespace.php:999]: It shouldn't be removed but it could potentially use different variable names. The variable names are suitably descriptive in both cases though. The first line gives the width and height to fit the image to, the second gives the width and height after being constrained to match the actual image dimensions.

Comment (jazzsequence) [reply inc/cropper/namespace.php:999]: Doesn't the second line effectively overwrite the `$width` and `$height` variables defined in the first line, though?

Comment (roborourke) [reply inc/cropper/namespace.php:999]: The entire function is overwriting those variables as it goes to reach the final valid version of them. Overwriting variables isn't inherently bad 😄  Would recommend watching back @tfrommen's workshop on performance, it has some really interesting points.
